src/gsi/setupoz.f90

```diff
@@ -1668,24 +1682,32 @@ subroutine contents_netcdf_diag_(odiag)
            call nc_diag_metadata("Observation",                  sngl(ozlv)                     ) 
            call nc_diag_metadata("Obs_Minus_Forecast_adjusted",  sngl(ozone_inv)                )
            call nc_diag_metadata("Obs_Minus_Forecast_unadjusted",sngl(ozone_inv)                )
-           call nc_diag_metadata("Reference_Pressure",           sngl(preso3l)                  )
+           call nc_diag_metadata("Reference_Pressure",           sngl(preso3l*r100)             ) ! Pa
+           if(luse(i)) then
+             call nc_diag_metadata("Analysis_Use_Flag",          one                            )
+           else
+             call nc_diag_metadata("Analysis_Use_Flag",          -one                           )
+           endif
            call nc_diag_metadata("Input_Observation_Error",      sngl(obserror)                 ) 
-           if(obstype =="omps_lp")then
+           if(obstype =="ompslp")then
              call nc_diag_metadata("Log10 Air Number Density",   sngl(airnd))
-             call nc_diag_metadata("Log10 Ozone Number Density UV", sngl(uvnd))
+             call nc_diag_metadata("Log10 Ozone Number Density UV",  sngl(uvnd))
              call nc_diag_metadata("Log10 Ozone Number Density VIS",sngl(visnd))
            endif
-
-           if(luse(i)) then
-             call nc_diag_metadata("Analysis_Use_Flag",            1             )
-           else
-             call nc_diag_metadata("Analysis_Use_Flag",            -1             )
+           call nc_diag_metadata("Forecast_adjusted", sngl(o3ppmv))
+           call nc_diag_metadata("Forecast_unadjusted", sngl(o3ppmv))
+          !if (wrtgeovals) then
+          !   ozgestmp = ozgestmp *constoz
+          !   call nc_diag_data2d("mole_fraction_of_ozone_in_air",  sngl(ozgestmp))
+          !   call nc_diag_data2d("air_pressure",sngl(exp(prsltmp)*r1000)) ! Pa
+          !endif
+           k1 = k
+           k2 = k - 1
+           if(k2 == 0)k2 = 1
+           if(k == nlevs)then
+             k1=nlevs-1
+             k2=1
            endif
-           if (save_jacobian) then
-              call nc_diag_data2d("Observation_Operator_Jacobian_stind", dhx_dx%st_ind)
-              call nc_diag_data2d("Observation_Operator_Jacobian_endind", dhx_dx%end_ind)
-              call nc_diag_data2d("Observation_Operator_Jacobian_val", real(dhx_dx%val,r_single))
-            endif
```

Comment:
@jack-woollen - I think this if-block that saves the Jacobian to the diag file should not be removed.  Do you agree?

Reply:
@jswhit I don't know. It looks like in line 600 they're writing the whole array instead of just a couple values. Is that a replacement for the deleted block?

                if (save_jacobian) then
                    call fullarray(dhx_dx, dhx_dx_array)
                    call nc_diag_data2d("Observation_Operator_Jacobian", dhx_dx_array)
                 endif

Reply:
no it's not. I will have to add the jacobian writes back in.